client/components/popover/menu.jsx

```diff
@@ -109,7 +115,15 @@ class PopoverMenu extends Component {
 		this._previouslyFocusedElement = document.activeElement;
 
 		if ( elementToFocus ) {
-			elementToFocus.focus();
+			// Defer the focus a bit to make sure that the popover already has the final position.
+			// Initially, after first render, the popover is positioned outside the screen, at
+			// { top: -9999, left: -9999 } where it already has dimensions. These dimensions are measured
+			// and used to calculate the final position.
+			// Focusing the element while it's off the screen would cause unwanted scrolling.
+			this.delayedFocus = setTimeout( () => {
+				elementToFocus.focus();
+				this.delayedFocus = null;
```

Comment:
I don't think setting `this.delayedFocus` to null makes sense here, the value returned by `setTimeout()` is just an `integer` key that can be passed to `clearTimeout` later.

Reply:
Setting it to `null` is to just prevent `clearTimeout()` from being called when the component unmounts. It might not be necessary though because I think `clearTimeout()` is probably resilient to being called with a timeout id that's already passed.